modules/effects/src/main/scala/dev/profunktor/redis4cats/redis.scala

```diff
@@ -42,22 +43,43 @@ import io.lettuce.core.{
   ZAddArgs,
   ZAggregateArgs,
   ZStoreArgs,
+  ExpireArgs => JExpireArgs,
   GetExArgs => JGetExArgs,
   Limit => JLimit,
   Range => JRange,
   ReadFrom => JReadFrom,
   ScanCursor => JScanCursor,
-  SetArgs => JSetArgs,
-  ExpireArgs => JExpireArgs
+  SetArgs => JSetArgs
 }
 import io.lettuce.core.api.async.RedisAsyncCommands
 import io.lettuce.core.cluster.api.async.RedisClusterAsyncCommands
 import io.lettuce.core.cluster.api.sync.{ RedisClusterCommands => RedisClusterSyncCommands }
+import org.typelevel.keypool.KeyPool
 
 import scala.concurrent.duration._
 
 object Redis {
 
+  object Pool {
+    final case class PoolSettings(maxTotal: Int, maxIdle: Int, idleTimeAllowedInPool: FiniteDuration)
+    object PoolSettings {
```

Comment:
The prefix is redundant, as it's already defined within the `Pool` enclosing object.

```suggestion
    object Settings {
```